### PR TITLE
readme: document metrics settings

### LINKED ISSUE #1446: documentation: explain settings.metrics.send-metrics
I forgot to document metricdog settings in the main Bottlerocket README. 

particularly `settings.metrics.send-metrics` 

Fix it!

### PR DESCRIPTION
**Pull request by webern**
**Issue number:**

Closes #1446

**Description of changes:**

Add metrics settings to the settings section of the main readme.

**Testing done:**

N/A

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -394,6 +394,16 @@ The no-proxy list will automatically include entries for localhost.
 
 If you're running a Kubernetes variant, the no-proxy list will automatically include the Kubernetes API server endpoint and other commonly used Kubernetes DNS suffixes to facilitate intra-cluster networking.
 
+#### Metrics settings
+
+By default, Bottlerocket sends anonymous metrics when it boots, and once every six hours.
+This can be disabled by setting `send-metrics` to false.
+Here are the metrics settings:
+
+* `metrics.metrics-url`: the endpoint to which metrics will be sent.
```
Comment (etungsten) [inline README.md]: super-duper-uber-minor nit: Can we capitalize the first word of the descriptions just to be consistent with the rest of the settings documentation?

Comment (etungsten) [inline README.md]: Maybe we should also mention that this `metrics-url` is set by default as well. And it's set to send to a Bottlerocket-team-controlled endpoint (not sure how to word this).

File: README.md [hunk 2]
```diff
@@ -394,6 +394,16 @@ The no-proxy list will automatically include entries for localhost.
...
+* `metrics.send-metrics`: whether or not Bottlerocket will send anonymous metrics.
+* `metrics.service-checks`: the systemd services that will be checked to determine whether a host is healthy or not.
```
Comment (etungsten) [inline README.md]:
```suggestion
* `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy or not.
```

**Comment by webern** [comment]
> webern force-pushed the webern:readme-metrics branch from c382f50 to 379bc29

Take @etungsten's suggestions.

**Review by etungsten — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -394,6 +394,16 @@ The no-proxy list will automatically include entries for localhost.
...
 
+#### Metrics settings
+
+By default, Bottlerocket sends anonymous metrics when it boots, and once every six hours.
+This can be disabled by setting `send-metrics` to false.
+Here are the metrics settings:
+
+* `metrics.metrics-url`: The endpoint to which metrics will be sent. The default is `https://metrics.bottlerocket.aws/v1/metics`.
+* `metrics.send-metrics`: Whether or not Bottlerocket will send anonymous metrics.
+* `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy or not.
```
Comment (bcressey) [inline README.md]: We typically prepend `settings.` to these values.

File: README.md [hunk 4]
```diff
@@ -394,6 +394,16 @@ The no-proxy list will automatically include entries for localhost.
 
 If you're running a Kubernetes variant, the no-proxy list will automatically include the Kubernetes API server endpoint and other commonly used Kubernetes DNS suffixes to facilitate intra-cluster networking.
 
+#### Metrics settings
+
+By default, Bottlerocket sends anonymous metrics when it boots, and once every six hours.
+This can be disabled by setting `send-metrics` to false.
+Here are the metrics settings:
+
+* `metrics.metrics-url`: The endpoint to which metrics will be sent. The default is `https://metrics.bottlerocket.aws/v1/metics`.
```
Comment (tjkirch) [inline README.md]:
```suggestion
* `metrics.metrics-url`: The endpoint to which metrics will be sent. The default is `https://metrics.bottlerocket.aws/v1/metrics`.
```

File: README.md (hunk 3 above)
Comment (tjkirch) [inline README.md]:
```suggestion
* `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy.
```
nit: "whether" usually implies "or not".  (same in send-metrics, if you don't hate this suggestion.)

**Comment by webern** [comment]
> webern force-pushed the webern:readme-metrics branch from 379bc29 to a5b1355

@bcressey and @tjkirch suggestions

**Review by tjkirch — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**